.gitignore

```diff
@@ -59,7 +55,6 @@ hs_err_pid*
 spark-warehouse/
 metastore_db
 derby.log
-ldap
```

Comment:
I don't see any temporary folder names `ldap`, but this exclude the `ldap` package from the source code folder